packages/discord.js/src/structures/Embed.js

```diff
@@ -6,31 +6,26 @@ const isEqual = require('fast-deep-equal');
  * Represents an embed.
  */
 class Embed {
-  /**
-   * Creates a new embed object
-   * @param {APIEmbed} data API embed data
-   * @private
```

Comment:
Do we want to document this constructor? Iirc we don't document others, like Channels for example 👀

Reply:
We can't document this as `@private` else it won't show in the sidebar in the documentation. If we publicly document it, well, they should be using `EmbedBuilder` instead.